src/main/scala/cromwell/Main.scala

```diff
@@ -51,8 +48,18 @@ object Main extends App {
   }
 
   def highlight(args: Array[String]) {
+    if (args.length >= 2 && !Seq("html", "console").contains(args(1))) {
+      println("usage: highlight <wdl> <html|console>")
+      System.exit(1)
+    }
+
+    val visitor = args match {
+      case a if a.length >= 2 && args(1) == "html" => HtmlSyntaxHighlighter
```

Comment:
it *could* be toggleable via config but I'm not sure it's worth doing it.  This particular path is the CLI anyway so it makes sense to get the highlighter from the command line args.

In other contexts, all you have to do is `new SyntaxFormatter(HtmlSyntaxHighlighter)` and having a default set in a config doesn't appear to me to add a lot of value and might even hurt because the line of code as I wrote it is more explicit.